jerry-core/ecma/builtin-objects/ecma-builtins.c

```diff
@@ -891,18 +891,16 @@ ecma_builtin_try_to_instantiate_property (ecma_object_t *object_p, /**< object *
  */
 void
 ecma_builtin_list_lazy_property_names (ecma_object_t *object_p, /**< a built-in object */
-                                       bool separate_enumerable, /**< true -  list enumerable properties into
-                                                                  *           main collection, and non-enumerable
-                                                                  *           to collection of 'skipped non-enumerable'
-                                                                  *           properties,
-                                                                  *   false - list all properties into main collection.
-                                                                  */
+                                       uint32_t opts, /**< listing options */
```

Comment:
Could we update the comment to reference the `ecma_list_properties_options_t` enum?
like: `uint32_t opts, /**< listing options using flags from ecma_list_properties_options_t */`

or maybe we should just mention it at the method's comments? What do you think?
(btw: this also applies to the other places where the `opts` is used)